src/transformers/integrations/integration_utils.py

```diff
@@ -1260,7 +1260,9 @@ def _initialize_run(self, **additional_neptune_kwargs):
         self._stop_run_if_exists()
 
         try:
-            self._run = init_run(**self._init_run_kwargs, **additional_neptune_kwargs)
+            run_params = additional_neptune_kwargs.copy()
+            run_params.update(self._init_run_kwargs)
+            self._run = init_run(**self._init_run_kwargs, **run_params)
```

Comment:
This looks to me like it won't work. Because we're updating `run_params` with `self._init_run_kwargs` we're guaranteeing that all the keys in `self._init_run_kwargs` are in `run_params` i.e. we're passing the same argument twice to `init_run`, which I'd expect to trigger an error.